### PR TITLE
➕ Added a README & a tool

### PR DESCRIPTION
**Pull request by ABSphreak**
- Added my README to Typing section
- Added Shields Project to Tools

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@
 - [Waylon Walker](https://github.com/WaylonWalker/WaylonWalker)
 
 #### Typing.. Mode 🎰
+- [Abhinav Sharma](https://github.com/ABSphreak/ABSphreak)
```
Comment (abhisheknaiidu) [inline README.md:28]: @ABSphreak Add it to the last in the category list, because the first one's are somewhat better :)

File: README.md [hunk 2]
```diff
@@ -38,6 +39,7 @@
 ## Tools
 - [Visitor Badge](https://visitor-badge.glitch.me/#docs) - Count visitors for your README.md, Issues, PRs in GitHub
 - [Simple Icons](https://github.com/simple-icons/simple-icons#cdn-usage) -  SVG icons for popular brands for your README.md files
+- [Shields Project](https://shields.io/) - Use Shields to create profile badges (Compatible with Simple Icons)
```
Comment (abhisheknaiidu) [inline README.md:42]: @ABSphreak This is a nice tool, I think adding this in 2nd place would be better, and also remove the brackets `(Compatible with Simple Icons)`

File: README.md (hunk 1 above)
Comment (ABSphreak) [reply README.md:28]: Oh, I thought it was alphabetical.

File: README.md (hunk 2 above)
Comment (ABSphreak) [reply README.md:42]: Even visitors badge can be created from here, this is a bigger project than that, anyway, I will add it to 2nd as you requested.

**Comment by ABSphreak** [comment]
And I think these should be like this, "One's"  → "Ones" & "GIF'S" → "GIFS"
I did not add it to my commit.

**Comment by abhisheknaiidu** [comment]
@ABSphreak Looks Great! I'm merging now.
 P.S - Don't forget to leave a ⭐ if you found this useful.
Thanks

**Pull request merged**